tests/features/post.feature

```diff
@@ -1,6 +1,12 @@
 Feature: Post
   Test the importing an post
 
+  Scenario: Article has correct number of images
+    Given the post "http://www.shortlist.com/style/fashion/best-trainers-sneakers-hottest-february-2017"
+    Then I should have an object of the post
+    And the post has the headline "The 10 Hottest Trainers for February 2017"
+    And the post has 10 images
```

Comment:
Bonus points for making this handy function you created a little more reusable, such as:
`And the post has 10 "image" widgets`, so you can then do `And the post has 2 "embed" widgets` etc.